app/experimenter/experiments/models.py

```diff
@@ -377,17 +381,38 @@ def has_normandy_info(self):
         return self.normandy_slug or self.normandy_id
 
     @property
-    def normandy_api_recipe_url(self):
-        if self.normandy_id:
-            return settings.NORMANDY_API_RECIPE_URL.format(id=self.normandy_id)
+    def format_dc_normandy_urls(self):
+        # returns a list of dictionaries containing D.C. and Normandy
+        # urls for the main normandy id and other normandy ids if they exist
+        normandy_recipe_url = settings.NORMANDY_API_RECIPE_URL
+        delivery_console_url = settings.DELIVERY_CONSOLE_RECIPE_URL
 
-    @property
-    def delivery_console_recipe_url(self):
         if self.normandy_id:
-            return settings.DELIVERY_CONSOLE_RECIPE_URL.format(
-                id=self.normandy_id
+            urls = []
+            urls.append(
+                {
+                    "id": self.normandy_id,
+                    "normandy_url": normandy_recipe_url.format(
+                        id=self.normandy_id
+                    ),
+                    "DC_url": delivery_console_url.format(id=self.normandy_id),
+                }
             )
 
+            if self.other_normandy_ids:
+                for id in self.other_normandy_ids:
+                    urls.append(
+                        {
+                            "id": id,
+                            "normandy_url": normandy_recipe_url.format(id=id),
+                            "DC_url": delivery_console_url.format(id=id),
+                        }
+                    )
+
+            return urls
+
+        return None
+
```

Comment:
at first glance it might be like... what's up with this property that makes this list of dicts.... but essentially I arrived at this because I needed a method that I could call in the template that could turn all of the `other_normandy_ids` into DC urls and normandy urls. Can't call a function and pass things into it in a django template very easily... so I made a property on the model that would essentially return a dict for each id, the main and extra ids if there are any... with both of the necessary urls... and then it made it pretty simple on the template side to loop over this list to display the ids and urls in the normandy info section....  perhaps someone has a better idea...

Reply:
Yeah I mean I do this kind of thing all the time, your instincts are correct here 👍

Reply:
Or at least if it's wrong, then we're both wrong together XD